packages/manager/modules/web-paas/src/details/service/add-addon/controller.js

```diff
@@ -0,0 +1,113 @@
+import debounce from 'lodash/debounce';
+import get from 'lodash/get';
+import moment from 'moment';
+
+export default class {
+  /* @ngInject */
+  constructor($q, $scope, WebPaas, $translate) {
+    this.$q = $q;
+    this.$scope = $scope;
+    this.WebPaas = WebPaas;
+    this.$translate = $translate;
+    this.disabled = false;
+  }
+
+  $onInit() {
+    this.currentMonth = moment().format('MMMM');
+    this.setParameters();
+    this.$scope.$watch(
+      '$ctrl.addon.quantity',
```

Comment:
We have added a debounce since we have to call the API to fetch prices. So we are pausing for a second before we call API, to make sure user has a smooth experience while increasing quantity quickly